inspectors/models/notation/clefs/clefsettingsmodel.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CLEFSETTINGSMODEL_H
+#define CLEFSETTINGSMODEL_H
+
+#include "models/abstractinspectormodel.h"
+
+class ClefSettingsModel : public AbstractInspectorModel
+{
+    Q_OBJECT
+
+    Q_PROPERTY(PropertyItem* showCourtesy READ showCourtesy CONSTANT)
```

Comment:
Usually, I prefer to name the bool properties using markers "is" and "has" - that's a popular practice within any language

In QML, "model.showCourtesy" may sound like a direct action - i.e. "Q_INVOKABLE void showCourtesy()"